autoconfigure/collector-sqs/src/test/java/zipkin/autoconfigure/collector/sqs/ZipkinSQSCollectorAutoConfigurationTest.java

```diff
@@ -25,19 +27,22 @@
 import org.springframework.context.annotation.Configuration;
 import zipkin.collector.CollectorMetrics;
 import zipkin.collector.CollectorSampler;
-import zipkin.collector.sqs.AwsSqsCollector;
+import zipkin.collector.sqs.SQSCollector;
+import zipkin.junit.aws.AmazonSQSRule;
 import zipkin.storage.InMemoryStorage;
 import zipkin.storage.StorageComponent;
 
 import static org.assertj.core.api.Assertions.assertThat;
-
 import static org.springframework.boot.test.util.EnvironmentTestUtils.addEnvironment;
 
-public class ZipkinSqsCollectorAutoConfigurationTest {
+public class ZipkinSQSCollectorAutoConfigurationTest {
 
   @Rule
   public ExpectedException thrown = ExpectedException.none();
 
+  @Rule
+  public AmazonSQSRule sqsRule = new AmazonSQSRule().start(9324);
```

Comment:
Now that we have a valid queueUrl populated from the ZipkinSQSCollectorProperties the AmazonSQSClient will throw an exception if its not able to connect to the URL.